modules/secrets.js

```diff
@@ -51,23 +55,39 @@ const SENTRY_DSN = getSecret('sentry.dsn');
 const AZURE_AUTH = {
     MS_CLIENT_ID: getSecret('ms.auth.tnlcf.clientId'),
     MS_CLIENT_SECRET: getSecret('ms.auth.tnlcf.clientSecret'),
-    MS_REDIRECT_URL: process.env.MS_REDIRECT_URL || getSecret('ms.auth.redirectUrl')
+    MS_REDIRECT_URL:
+        process.env.MS_REDIRECT_URL || getSecret('ms.auth.redirectUrl')
+};
+
+/**
+ * Salesforce authentication
+ */
+const SALESFORCE_AUTH = {
```

Comment:
This will come from parameter store once we've confirmed the API user we are connecting with.